Cargo.toml

```diff
@@ -25,5 +25,8 @@ tabled = "0.7.0"
 thiserror = "1.0"
 tokio = { version = "1.18", features = ["rt-multi-thread", "macros"] }
 
+[dependencies.serde_with]
+version = "2.0.0"
```

Comment:
For the sake of consistency, could you please write this like other dependencies?
```
serde_with = "2.0.0"
```